src/ildasm/dasm.cpp

```diff
@@ -3700,9 +3700,9 @@ lDone: ;
                 }
             }// end for( along the argnames)
 #ifdef _WIN64
-                sprintf_s(szArgPrefix,32,"@%I64d0",(size_t)pszArgname);
+                sprintf_s(szArgPrefix,MAX_PREFIX_SIZE,"@%I64d0",(size_t)pszArgname);
```

Comment:
A nit - since you are touching this code, you could remove the ifdef and use %I instead. %I is 32 bit on 32 bit platforms and 64 bits on 64 bit platforms.